service/src/main/resources/db/changesets/20220923_initial.yaml

```diff
@@ -0,0 +1,21 @@
+databaseChangeLog:
+  - changeSet:
+      id: initial
+      author: ishaghag
+      changes:
+        - createTable:
+            tableName: user_profile
+            columns:
+              - column:
+                  name: user_id
+                  type: text
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+                  remarks: A persistent user ID retrieved from SAM
+              - column:
+                  name: profile_obj
```

Comment:
Just curious why you want this to be jsonb. Why not have a "recently used workspaces" column, starred workspaces column, etc.

Reply:
+1 
How does using an object (with internal fields) compare with a flat schema where some of the critical user properties are their own columns?

For example --> 

user_id text
spend_profile_id text // example only
primary_workspace text. // just an example, include baseline properties here
additional_properties object // custom properties that we may use in the future

Reply:
The intent is to be schemaless (per the doc), with the benefit being that a service update isn't required to store a new field.

Reply:
Oh yeah, I forgot about that.

nit To avoid "obj" abbreviation, how about `user_profile`, `profile`, or `profile_object` ?

Reply:
outside the scope of this PR - Is no-sql DB under consideration to go schema-less instead of using json/jsonB fields within sql